Fix ambiguity around ROUTE_ADVERTISEMENT (#142)
ietf-wg-masque/draft-ietf-masque-connect-ip: draft-ietf-masque-connect-ip.md

capsule MUST follow the error handling procedure defined in
{{Section 3.3 of HTTP-DGRAM}}.

Upon receiving the ROUTE_ADVERTISEMENT capsule, an endpoint MAY install

> corresponding entries into its routing table.

Reviewer: In many cases, won't this be equivalent? If I don't have a route to send on, I generally won't be able to send.

I also don't love changing the text to imply that a routing table is used by an implementation to provide access to the CONNECT-IP forwarding session. That's sometimes the case, but there are other models (like just having a flow-scoped connection). Elsewhere, we mention routing tables only as examples, not as a normative behavior.

Reviewer: ```suggestion
Upon receiving the ROUTE_ADVERTISEMENT capsule, an endpoint MAY update its local
state about what its peer is willing to route (subject to local policy), such as by installing
entries in a routing table.
```

capsule MUST follow the error handling procedure defined in
{{Section 3.3 of HTTP-DGRAM}}.

Upon receiving the ROUTE_ADVERTISEMENT capsule, an endpoint MAY update its local

> state about what its peer is willing to route (subject to local policy), such

Author: ```suggestion
state regarding what its peer is willing to route (subject to local policy), such
```

> state about what its peer is willing to route (subject to local policy), such as by installing
> entries in a routing table.

Author: ```suggestion
state about what its peer is willing to route (subject to local policy), such
as by installing entries in a routing table.
```
